### ISSUE TITLE
Configurable Data Augmentation UI to prevent model overfitting

### ISSUE THREAD

**Issue by Amith102** [Sree14hari/EPOQ#8]
When users upload smaller datasets, models are highly prone to overfitting (where training accuracy is high but validation accuracy is low). Currently, users have to manually augment their images before importing them into EPOQ.

**Comment by Amith102** [comment]
I would like to work on this issue as part of OSQ
